src/customers/components/CustomerAddress/CustomerAddress.tsx

```diff
@@ -99,6 +96,20 @@ const CustomerAddress: React.FC<CustomerAddressProps> = props => {
                   description: "button"
                 }),
                 onSelect: () => onSetAsDefault(AddressTypeEnum.BILLING)
+              },
+              {
+                label: intl.formatMessage({
```

Comment:
We may even move all messages into separate `const messages = defineMessages({...})`